backend/characterconnect/api/game_manager.py

```diff
@@ -1,7 +1,9 @@
 from __future__ import annotations
+from ctypes.wintypes import CHAR
 from typing import Dict, List, MutableSet, Tuple, Optional
 import random
 import dataclasses
+from xmlrpc.client import boolean
```

Comment:
This should be removed, the inbuilt `bool` type in Python is what you're looking for.